### ISSUE TITLE
RCF API Test

### ISSUE THREAD

**Issue by Futabay** [NASA-AMMOS/AIT-DSN#128]
1. SSPSim:
i. Go to SSPSim website
ii. Open "MISSION MANAGERS" >  "Test" > "RCFv2 ONLC2"
iii. Ensure that the Production Id is set to "TestBaseband2"
iv. Ensure that GVCID is set to (250, 0, *) 
v. Open "PRODUCTION" > "TestBaseband2"
vi. Ensure that Spacecraft Id is set to 250 (you can set SCID by clicking at top-right corner gear icon)
vii. Ensure that VCID is checked and set to 6 (you can set VCID by clicking at top-right corner gear icon)
viii. Activate the TM data flow by clicking the green arrow labelled "TM"
ix. Activate the TM simulation data creation by clicking the green arrow labelled "SIM".
x. Activate the Mission Manager interface by clicking the green arrow labelled "SVC"

2. Run AIT-DSN/ait/dsn/bin/examples/rcf_api_test.py:
`python rcf_api_test.py`

   Note: changed AIT-DSN/ait/dsn/sle/rcf.py line 434 and 438 to `ait.core.log.info`

3. Stop AIT-DSN/ait/dsn/bin/examples/rcf_api_test.py by Ctl+C

4. Verify each communication status are successful as shown terminal log below.

SSPSim - Configuration Setting
![Screen Shot 2020-08-21 at 3 47 21 PM](https://user-images.githubusercontent.com/11483503/90941616-848c3680-e3c7-11ea-88b9-bdf1951a09dc.png)

SSPSim - Running AIT-DSN/ait/dsn/bin/examples/rcf_api_test.py 1
![Screen Shot 2020-08-21 at 3 55 40 PM](https://user-images.githubusercontent.com/11483503/90941433-f57f1e80-e3c6-11ea-8f3c-50c433400acc.png)

SSPSim - Running AIT-DSN/ait/dsn/bin/examples/rcf_api_test.py 2
![Screen Shot 2020-08-21 at 3 55 47 PM](https://user-images.githubusercontent.com/11483503/90941471-18113780-e3c7-11ea-8a4e-fd51032637a4.png)

Stop AIT-DSN/ait/dsn/bin/examples/raf_api_test.py by Ctl+C
![Screen Shot 2020-08-21 at 3 56 09 PM](https://user-images.githubusercontent.com/11483503/90941518-324b1580-e3c7-11ea-9cf5-181bcb087878.png)

SSPSim - Log
![Screen Shot 2020-08-21 at 3 56 16 PM](https://user-images.githubusercontent.com/11483503/90941576-5c9cd300-e3c7-11ea-914c-e1ad631f7b73.png)

Terminal Log
```
(ait) LMC-062858:examples kkallas$ python rcf_api_test.py
2020-08-21T15:55:33.111 | WARNING  | Config parameter command.history.filename specifies nonexistent path /Users/kkallas/Documents/github/AIT/cmdhist.pcap
2020-08-21T15:55:33.113 | WARNING  | Config parameter dsn.cfdp.mib.path specifies nonexistent path /Users/kkallas/Documents/github/AIT/AIT-Core/config/mib
2020-08-21T15:55:33.114 | WARNING  | Config parameter dsn.cfdp.datasink.outgoing.path specifies nonexistent path /Users/kkallas/Documents/github/AIT/AIT-Core/ait/dsn/cfdp/datasink/outgoing
2020-08-21T15:55:33.115 | WARNING  | Config parameter dsn.cfdp.datasink.incoming.path specifies nonexistent path /Users/kkallas/Documents/github/AIT/AIT-Core/ait/dsn/cfdp/datasink/incoming
2020-08-21T15:55:33.116 | WARNING  | Config parameter dsn.cfdp.datasink.tempfiles.path specifies nonexistent path /Users/kkallas/Documents/github/AIT/AIT-Core/ait/dsn/cfdp/datasink/tempfiles
2020-08-21T15:55:33.117 | WARNING  | Config parameter dsn.cfdp.datasink.pdusink.path specifies nonexistent path /Users/kkallas/Documents/github/AIT/AIT-Core/ait/dsn/cfdp/datasink/pdusink
2020-08-21T15:55:33.118 | WARNING  | Config parameter data.1553.path specifies nonexistent path /gds/dev/data/LMC-062858/2020/2020-234/downlink/1553
2020-08-21T15:55:33.118 | WARNING  | Config parameter data.bad.path specifies nonexistent path /gds/dev/data/LMC-062858/2020/2020-234/downlink/bad
2020-08-21T15:55:33.119 | WARNING  | Config parameter data.lehx.path specifies nonexistent path /gds/dev/data/LMC-062858/2020/2020-234/downlink/lehx
2020-08-21T15:55:33.120 | WARNING  | Config parameter data.planning.path specifies nonexistent path /gds/dev/data/LMC-062858/2020/2020-234/planning
2020-08-21T15:55:33.121 | WARNING  | Config parameter data.sdos.path specifies nonexistent path /gds/dev/data/LMC-062858/2020/2020-234/sdos
2020-08-21T15:55:33.122 | WARNING  | Config parameter data.uplink.path specifies nonexistent path /gds/dev/data/LMC-062858/2020/2020-234/uplink
2020-08-21T15:55:33.123 | WARNING  | Config parameter data.ats.path specifies nonexistent path /gds/dev/data/LMC-062858/2020/2020-234/ats
2020-08-21T15:55:33.457 | INFO     | Connection to DSN successful through atb-ocio-sspsim.jpl.nasa.gov.
2020-08-21T15:55:33.458 | INFO     | Configuring SLE connection...
2020-08-21T15:55:33.459 | INFO     | SLE connection configuration successful
2020-08-21T15:55:35.465 | INFO     | Sending Bind request ...
2020-08-21T15:55:35.523 | INFO     | Bind successful
2020-08-21T15:55:37.471 | INFO     | Sending data start invocation ...
2020-08-21T15:55:37.516 | INFO     | Start successful
2020-08-21T15:55:38.151 | INFO     | Sending <class 'ait.dsn.sle.frames.TMTransFrame'> with 1275 bytes to frame port
2020-08-21T15:55:38.892 | INFO     | Sending <class 'ait.dsn.sle.frames.TMTransFrame'> with 1275 bytes to frame port
2020-08-21T15:55:39.540 | INFO     | Sending <class 'ait.dsn.sle.frames.TMTransFrame'> with 1275 bytes to frame port
...
2020-08-21T15:55:50.559 | INFO     | Sending <class 'ait.dsn.sle.frames.TMTransFrame'> with 1275 bytes to frame port
^C2020-08-21T15:55:51.198 | INFO     | Sending data stop invocation ...
2020-08-21T15:55:51.248 | INFO     | Stop successful
2020-08-21T15:55:53.203 | INFO     | Sending Unbind request ...
2020-08-21T15:55:53.240 | INFO     | Unbind successful
```

**Comment by Futabay** [comment]
Found bugs:
* byte string conversion issue - issue #118

**Comment by MJJoyce** [comment]
Resolved in #120
